aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java

```diff
@@ -306,7 +307,15 @@ public final class AotMachine implements Machine {
                     .shared(OpCode.F64_TRUNC, OpcodeImpl.class)
                     .build();
 
+    public static Function<Instance, Machine> withClassName(String className) {
```

Comment:
The reason to allow setting the class name is for actual ahead-of-time purposes. Users of `AotMachine` today will never need to set this, so I don't think we need this factory.

Reply:
It was just appealing to add it ... if you have a strong opinion, I have no problems removing it as it's just cosmetics :-)

Reply:
I just don't think it's useful as it won't be used outside of tests. It seems to encourage users to use it, even though it doesn't do anything.

Reply:
ok, removed :+1: